docs/UserGuide.md

```diff
@@ -60,6 +60,16 @@ level | Refers to the education level of a student in the context of Singapore's
 
 ### Application layout
 
+![gui_layout.png](gui_layout.png)
+
+The GUI of Track-O is divided mainly into 4 sections:
+- `Menu Bar`: Navigation buttons that you can click to exit Track-O or open up a window containing a link to this User Guide. 
+- `Tutee list`: A scrollable panel that shows a numbered list of your tutees.
+  - The index number of the tutee list is the number that is assigned to each tutee in the numbered list.
+- `Command box`: A text box where you can type in commands and hit `enter` to execute them.
```

Comment:
Just wondering if we should use Enter or `enter` for this to stay consistent with the rest of the UG.